modules/ROOT/pages/enterprise/server/configure.adoc

```diff
@@ -194,10 +198,12 @@ ephox {
 }
 ----
 
+[[troubleshootingorigins]]
 ==== Troubleshooting Origins
 
-If you missed an Origin or specified an Origin incorrectly, {productname} features that rely on the server-side components will not work from that Origin. If you observe that requests to the server-side components are failing or features are unavailable and you're not sure why, refer to the troubleshooting information about link:{baseurl}/enterprise/server/troubleshoot/#usingbrowsertoolingtoinvestigateservicesissues[Using browser tooling to investigate services issues].
+If you missed an Origin or specified an Origin incorrectly, {productname} features that rely on the server-side components will not work from that Origin. If you observe that requests to the server-side components are failing or features are unavailable and you're not sure why, refer to the troubleshooting information about xref:enterprise/server/troubleshoot.adoc#usingbrowsertoolingtoinvestigateservicesissues[Using browser tooling to investigate services issues].
 
+[[proxyoptional]]
 === `proxy` (optional)
 
```

Comment:
In the table below theres some content that's being treated as markup. In this case it's the following line which is treating the last 2 asterisks as being markup for bold content:
```
| A list of strings separated by vertical lines ("\|") listing hosts and domains to be excluded from proxying, for *both* plain HTTP and HTTPS connections. The strings can contain asterisks ("*") as wildcards. (Optional, defaults to "localhost\|127.*\|[::1]" if not set.)
```

Reply:
done